### DISCUSSION TITLE
I'd like to improve your AMR package

### DISCUSSION THREAD

**Discussion by AndersonRodr** [msberends/AMR#174] [Ideas]
Hello Mr. Berends, How are you doing?
I am Anderson Rodrigues Cavalcante, a master's degree student from the Federal Rural University of Pernambuco - Brazil, and your AMR package has helped me a lot.
My research aims to predict antimicrobial resistance and I'm trying to use ARIMA to do that. However, my time series is poor and scarce. There are gaps and few amounts. 
Anyway, a contribution that can give me a good credit is work on your package, maybe including some Machine Learning Models, beyond the Logistic Regression model that already exists.

What do you think? I'd like to contribute and maybe be the first Brazilian on the list of contributing countries.
Thanks.

**Comment by msberends** [comment]
Dear Anderson,

I moved this to the 'Discussion' tab. 

Thank you for reaching out and for your kind words about the AMR package. I’m glad to hear it has been helpful in your research!

Your idea of expanding the package with additional machine learning models is intriguing. However, I would need more specific details about the kind of contributions you envision. What models or methods do you have in mind, and how do you see them aligning with the existing scope of the package?

Collaborations are always welcome, but it’s important to ensure that any new functionality adds substantial value to the package while adhering to its standards. I’d appreciate it if you could provide a more detailed proposal, including specific use cases or even a prototype, so we can assess how best to proceed.

Looking forward to hearing more about your ideas.

Best regards,
Matthijs

**Reply by AndersonRodr** [reply]
Hello, Mr. Berends. Thank you for replying to me.
My idea is to add a model, like ARIMA or Random Forest, to increase the options of your application.
